bot.py

```diff
@@ -43,6 +44,53 @@ async def on_member_join(member):
     await member.dm_channel.send(f"Hi {member.name}, welcome to 관전남 월드!")
 
 
+# remove default help command to use our help command
+bot.remove_command("help")
+
+# Custom help command
+@bot.command(name="help", help="This is help command")
+async def help_command(ctx):
+    """Help command outputs description about all the commands"""
+    # Create to display help command in embedded format
+    help_embed = discord.Embed(
+        title=f"How to use {bot.user.name}",
+        description=f"<@!{bot.user.id}> <command>",
+        color=discord.Color.gold(),
+    )
+
+    # ADD thumbnail (Image can be changed whatever we want. eg.our logo)
+    help_embed.set_thumbnail(url="https://emoji.gg/assets/emoji/3907_lol.png")
+
+    help_embed.add_field(name="** **", value="** **", inline=False)
+
+    help_embed.add_field(
+        name="** **",
+        value="**The list of command examples**",
+        inline=False,
+    )
+
+    help_embed.add_field(
+        name="** **",
+        value=f"<@!{bot.user.id}> **{help_command.name}** \n {help_command.help}",
+        inline=False,
+    )
+
+    help_embed.add_field(
+        name="** **",
+        value=f"<@!{bot.user.id}> **{get_rank.name} [summoner name]** \n {get_rank.help}",
+        inline=False,
+    )
+
+    help_embed.add_field(
+        name="** **",
+        value=f"<@!{bot.user.id}> **{get_last_match.name} [summoner name]** \n \
+         {get_last_match.help}",
+        inline=False,
+    )
```

Comment:
For these commands, we should make array of commands function -> loop through them instead of hardcoding it one by one. 

eg; list = [help, get_rank, get_last_match] <- loop through this array and add it to embed.

Reply:
I changed it